oauthproxy.go

```diff
@@ -457,6 +457,20 @@ func (p *OAuthProxy) ErrorPage(rw http.ResponseWriter, code int, title string, m
 	p.templates.ExecuteTemplate(rw, "error.html", t)
 }
 
+//<!--COCO Begin-->
+// COCO: logout from cocostore
+//<!--COCO End-->
+func (p *OAuthProxy) CocostoreLogout(rw http.ResponseWriter, title string) {
+        rw.WriteHeader(http.StatusOK)
+        t := struct {
```

Comment:
- add enough comments so that it is understood when somebody else looks at the code as we are not familiar with go code

Reply:
handled